plugins/default_web.py

```diff
@@ -37,12 +37,10 @@ def check_url(url):
             log.debug('trying ' + url)
             try:
                 with urllib.request.urlopen(url) as response:
-                    if response.getcode() != 200:
-                        log.debug("Bad response at %s (ignored)" % url)
-                        return None
```

Comment:
hm.... I don't follow you here. The http operation might be perfectly OK while returning something different than 200.  Should we not just discard that reply then? Or are you saying the that we can trust urllib to raise a HTTPError in all cases the code is != 200?

Reply:
Hm... reading up on urllib this ineed seems to be the case.